packages/theme/components/FiltersSidebar.vue

```diff
@@ -103,16 +130,25 @@ export default {
     SfFilter,
     SfAccordion,
     SfColor,
-    SfHeading
+    SfHeading,
+    SfRange
   },
   setup(props, context) {
-    const { changeFilters, isFacetColor } = useUiHelpers();
+    const { changeFilters, isFacetColor, isFacetPrice, getSearchPriceFromUrl} = useUiHelpers();
     const { toggleFilterSidebar, isFilterSidebarOpen } = useUiState();
     const { result } = useFacet();
+    const selectedPrice = ref({});
+
+    let range = [0, 300];
+    const urlPriceRange = getSearchPriceFromUrl();
+    if (typeof urlPriceRange !== 'undefined') {
```

Comment:
It could, but we would have to check if its undefined twice or hardcode a range inside getSearchPriceFromUrl

Reply:
Where would the checks be?

Reply:
inside UseUiHelpers would happen first check (url query.priceFilter can be undefined or a string), since we cannot split  undefined and we need to split query.priceFilter to obtain an array. 2nd one would happen in exactly the same place as it is now since before mentioned getSearchPriceFromUrl would return undefined or expected array(unless we hardcode getSearchPriceFromUrl to return a set array if url query.priceFilter is undefined )